endorser/api/core/config.py

```diff
@@ -83,6 +83,7 @@ class GlobalConfig(BaseSettings):
 
     ACAPY_ADMIN_URL: str = os.environ.get("ACAPY_ADMIN_URL", "http://localhost:9031")
     ACAPY_ADMIN_URL_API_KEY: str = os.environ.get("ACAPY_API_ADMIN_KEY", "change-me")
+    ACAPY_WALLET_AUTH_TOKEN: str = os.environ.get("ACAPY_WALLET_AUTH_TOKEN","")
```

Comment:
I think you just want `ACAPY_WALLET_AUTH_TOKEN: str = os.environ.get("ACAPY_WALLET_AUTH_TOKEN")` so the value defaults to `None` (rather than an empty string) when not provided.
Also I recommend including the `Bearer ` string in the code, so you just provide the token in the environment variable (rather than having to set the env var to `Bearer ...`).

Reply:
@ianco made the changes as suggested, please review, thanks.